pom.xml

```diff
@@ -9,7 +9,7 @@
   <properties>
     <java.level>8</java.level>
     <animal.sniffer.version>1.14</animal.sniffer.version>
-    <jenkins.version>1.609.3</jenkins.version>
+    <jenkins.version>2.187</jenkins.version>
```

Comment:
Not going to work that way. It should support few recent LTS baselines at least. I am not sure what is @omehegan 's preference, but I kindly request 2.150.x at least